cluster/aws/templates/format-disks.sh

```diff
@@ -42,39 +42,68 @@ for ephemeral_device in $ephemeral_devices; do
   fi
 done
 
+# These are set if we should move where docker/kubelet store data
+# Note this gets set to the parent directory
+move_docker=""
+move_kubelet=""
+
 # Format the ephemeral disks
 if [[ ${#block_devices[@]} == 0 ]]; then
   echo "No ephemeral block devices found"
 else
   echo "Block devices: ${block_devices}"
 
-  apt-get install --yes btrfs-tools
+  if [[ ${FILESYSTEM:-} == "btrfs" ]]; then
+    apt-get install --yes btrfs-tools
 
-  if [[ ${#block_devices[@]} == 1 ]]; then
-    echo "One ephemeral block device found; formatting with btrfs"
-    mkfs.btrfs -f ${block_devices[0]}
+    if [[ ${#block_devices[@]} == 1 ]]; then
+      echo "One ephemeral block device found; formatting with btrfs"
+      mkfs.btrfs -f ${block_devices[0]}
+    else
+      echo "Found multiple ephemeral block devices, formatting with btrfs as RAID-0"
+      mkfs.btrfs -f --data raid0 ${block_devices[@]}
+    fi
+    mount -t btrfs ${block_devices[0]} /mnt
+    move_docker="/mnt"
+    move_kubelet="/mnt"
+  elif [[ ${FILESYSTEM:-} == "ext4" ]]; then
+    if [[ ${#block_devices[@]} == 1 ]]; then
+      echo "One ephemeral block device found; formatting with ext4"
+      mkfs -t ext4 ${block_devices[0]}
+    else
+      # TODO: Use LVM?
```

Comment:
Yes, I think so.  Otherwise this is rather confusing that you only use the first device?